src/components/Work/Tabs/Cite.js

```diff
@@ -1,14 +1,18 @@
-import React from "react";
+import React, { useState } from "react";
 import PropTypes from "prop-types";
 import MetadataDisplay from "../MetadataDisplay";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 
 const styles = {
   monoSpace: {
     fontFamily: "monospace"
   },
   tabContent: {
     padding: "0 1rem"
-  }
+  },
+  copyLink: { width: "2rem" },
+  active: { color: "green" }
```

Comment:
Can we make sure to use the "Northwestern green", defined in `src/styles/sass/base/_variables.scss`.  Let's use `$dark-green`, line 51.